vue-tags-input/vue-tags-input.vue

```diff
@@ -140,6 +140,7 @@
       class="autocomplete"
       @mouseout="selectedItem = null"
       v-if="autocompleteOpen">
+      <slot name="autocomplete-header"></slot>
```

Comment:
The names of all other slots are in camelCase, so could you please change it to avoid inconsistent behaviour?

Reply:
Sure and corrected!